Script/dungeons/GloomyCaves.dat

```diff
@@ -269,7 +269,7 @@ Dungeon ELPURI_CAVE;
           e == Random { MinPrice = 50; MaxPrice = 10000; Category = FOOD|POTION; }
           u == Random { MinPrice = 200; MaxPrice = 10000; Category = WAND|TOOL; }
           r == Random { MinPrice = 200; MaxPrice = 10000; Category = SCROLL|BOOK; }
-          s == wand(WAND_OF_STRIKING);
```

Comment:
That's a big change from what players are used to. Maybe leave them both on the same square. Players might not opt to go to the black market and they will still require ascension gear like this wand.